pallets/schemas/src/mock.rs

```diff
@@ -53,7 +53,7 @@ impl WeightToFeePolynomial for WeightToFee {
 impl pallet_schemas::Config for Test {
 	type Event = Event;
 	type WeightInfo = ();
-	type MinSchemaModelSizeBytes = ConstU32<5>;
+	type MinSchemaModelSizeBytes = ConstU32<15>;
```

Comment:
Just curious, what does `15` represent here?

Reply:
the number of bytes of string representation of the schema model to be registered